src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

```diff
@@ -106,9 +106,9 @@ If you need to install the agent manually, follow these steps:
   <Step>
     ### (Android-only) Configure your Android app [#configure-android]
 
-    To configure your Android app you need to customize your Gradle templates. If you're using Unity 2019 or later, complete the following steps:
+    To configure your Android app you need to customize your Gradle templates. If you're using Unity 2019 or later, follow the steps below.
 
-    1. Include the New Relic Maven repository URL in the Gradle build settings. To do this, open your `mainTemplate.gradle` file. This file is usually located in the `Assets/Plugins/Android` folder. Now, open the New Relic Maven URL like this:
+    1. In your Gradle build settings, you'll need to include the New Relic Maven repository URL. To do this, open your `mainTemplate.gradle` file, which is usually located in the `Assets/Plugins/Android` folder. Add the following snippet:
```

Comment:
Should it be possible to add the link to this repository?

Reply:
That's a great catch! Let me get it from the SME

Reply:
@ndesai-newrelic Can you provide what the repo URL should be?